tests/integration/cli/test_export_inference.py

```diff
@@ -230,15 +226,21 @@ def run_cli_test(test_recipe: str, checkpoint_path: str, work_dir: Path, device:
         exported_model_path,
     ]
 
-    run_main(command_cfg=command_cfg, open_subprocess=fxt_open_subprocess)
+    run_main(
+        command_cfg=command_cfg,
+        open_subprocess=False if task in ("visual_prompting", "zero_shot_visual_prompting") else fxt_open_subprocess,
```

Comment:
@yunchu I temporarily added it not to open subprocess, is it fine?

Reply:
Please revert it and add xfail mark if needed as discussed in the Teams chat

Reply:
Done at https://github.com/openvinotoolkit/training_extensions/pull/3040/commits/e3d29aecd9e0d68544197beb6793bf3f486ff96e.